website/docs/cdktf/concepts/assets.mdx

```diff
@@ -200,8 +200,9 @@ namespace MyCompany.MyApp
 package main
 
 import (
-	"github.com/hashicorp/terraform-cdk/examples/go/documentation/generated/hashicorp/aws"
-	"github.com/hashicorp/terraform-cdk/examples/go/documentation/generated/hashicorp/aws/s3"
+	"github.com/hashicorp/terraform-cdk/examples/go/documentation/generated/hashicorp/aws/provider"
```

Comment:
This snippet is generated – see the `#NEXT_CODE_BLOCK_SOURCE` comment above the `CodeTabs` component this snippet is in.